x-pack/plugins/ml/public/application/settings/filter_lists/edit/edit_filter_list.js

```diff
@@ -371,6 +374,7 @@ export class EditFilterListUI extends Component {
                       canCreateFilter === false
                     }
                     fill
+                    data-test-subj={'mlSaveFilterListButton'}
```

Comment:
To be consistent with other naming, I'd suggest to make this `mlFilterListSaveButton`.